stix_shifter_modules/secretserver/stix_transmission/api_client.py

```diff
@@ -89,7 +99,9 @@ def build_searchId(self):
             self.search_id = s_id
         return s_id
 
-    def get_search_results(self, search_id, index_from=None, fetch_size=None):
+    def get_search_results(self, search_id, index_from=1, fetch_size=2):
+        ind_from = index_from
+        fet_size = fetch_size
```

Comment:
not sure what's the purpose of `ind_from` and `fet_size` and why they are assigned 1 and 2 values respectively.